signal_processing/feature_extraction.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+import neurokit2 as nk
+from math import log10
+from scipy import signal
+
+
+# Use pan tompkins algorithm to separate signal into n beats
+def apply_pan_tompkins(x, n_beats=110, fs=1000):
```

Comment:
Can we add a parameter to specify range around R peak which we want to collect? That one paper did 750, but another did 512, so it would be easier if this was a parameter that we could modify. Something like samples_to_the_right and samples_to_the_left